units.cfg

```diff
@@ -1,16 +1,16 @@
 #textdomain wesnoth-Flight_Freedom
-[race]
-    id=boat
-    male_name= _ "race^Boat"
-    female_name= _ "race+female^Boat"
-    plural_name= _ "race^Boats"
-    description= _"Boats are aquatic vessels used for a variety of purposes, from fishing to transport to warfare. They come in many sizes, ranging from small rowboats used for river transport to massive warships equipped with multiple ballistas and catapults. They are used and constructed almost exclusively by humans, although some intrepid elves, most notably Meneldur of Elensefar, have been known to set sail on human-built craft as well."
-    num_traits=1
-    undead_variation=null
-    ignore_global_traits=yes
-    {TRAIT_MECHANICAL}
-    {HUMAN_NAMES}
-[/race]
+#[race]
+#    id=boat
+#    male_name= _ "race^Boat"
+#    female_name= _ "race+female^Boat"
+#    plural_name= _ "race^Boats"
+#    description= _"Boats are aquatic vessels used for a variety of purposes, from fishing to transport to warfare. They come in many sizes, ranging from small rowboats used for river transport to massive warships equipped with multiple ballistas and catapults. They are used and constructed almost exclusively by humans, although some intrepid elves, most notably Meneldur of Elensefar, have been known to set sail on human-built craft as well."
+#    num_traits=1
+#    undead_variation=null
+#    ignore_global_traits=yes
+#    {TRAIT_MECHANICAL}
+#    {HUMAN_NAMES}
+#[/race]
```

Comment:
Again, I'd prefer ifdef-ing this out, or removing it entirely, instead of commenting it out.